.tmp/tasks/task-8.8-plugin-cron-execution.md

```
@@ -0,0 +1,746 @@
+    "success": 95,
+    "failed": 5,
+    "running": 0,
+    "pending": 0,
+    "avgEventsProcessed": 38.5,  // Calculated from 95 successful runs only (SUM(eventsProcessed) / 95)
+    "avgDuration": 120000  // Calculated from 95 successful runs only (SUM(duration_ms) / 95), in milliseconds
+  }
+}
+```
+
+**Note**: `avgEventsProcessed` and `avgDuration` are computed only from successful runs (`WHERE status = 'success' AND completedAt IS NOT NULL`). If there are no successful runs, both values return `0`.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)